specification/web/resource-manager/Microsoft.Web/stable/2019-08-01/WebApps.json

```diff
@@ -3323,14 +3323,14 @@
         }
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}/listsecrets": {
-      "post": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}/keys/{keyName}": {
+      "put": {
         "tags": [
           "WebApps"
         ],
-        "summary": "Get function secrets for a function in a web site, or a deployment slot.",
-        "description": "Description for Get function secrets for a function in a web site, or a deployment slot.",
-        "operationId": "WebApps_ListFunctionSecrets",
+        "summary": "Add or update a function secret.",
```

Comment:
if path name changed `secret` to `key`, description,. operationId should be changed also for consistent.

Reply:
No paths or routes were changed in this PR. Please see above.